### ISSUE TITLE
errorformat is missing column for errorformat

### ISSUE THREAD

**Issue by lewis6991** [derekwyatt/vim-scala#147]
From sbt.vim:
```
CompilerSet errorformat=
      \%E\ %#[error]\ %f:%l:\ %m,%C\ %#[error]\ %p^,%-C%.%#,%Z,
      \%W\ %#[warn]\ %f:%l:\ %m,%C\ %#[warn]\ %p^,%-C%.%#,%Z,
\%-G%.%#
```

Doesn't correctly parse as it is missing the line column.

This seems to correct the issue for me:
```
CompilerSet errorformat=
      \%E\ %#[error]\ %f:%l:%c:\ %m,%C\ %#[error]\ %p^,%-C%.%#,%Z,
      \%W\ %#[warn]\ %f:%l:%c:\ %m,%C\ %#[warn]\ %p^,%-C%.%#,%Z,
\%-G%.%#
```

This is using scala `2.12` with sbt `1.0`

I'm guessing a recent version of scala or sbt introduced the column numbers?

**Comment by reidrac** [comment]
This works for me too, scala `2.12` and sbt `1.0`. Thanks!

**Comment by lewis6991** [comment]
Resolved in 971ac9ab3fe945105ef88587cfe5273fa2c8e988
